GitCommands/Settings/AppSettings.cs

```diff
@@ -1247,6 +1247,18 @@ public static ThemeId ThemeId
             }
         }
 
+        public static string[] ThemeVariations
```

Comment:
??? Can the variation be handled as a new theme instead?
It can be confusing if a theme has no colrorblindness variation, this will have no effect. 
It will be easier to add variants of exiting themes too, like dark vs darksilver (most fields differe there though).
I do not want to overwork this functionality, but "automatic variations" is slightly better in my opinion.

Reply:
The point is to avoid having colorblindness as another theme, so that we do not end up with 6 themes instead of 3 (bright, dark, darksilver).

> It can be confusing if a theme has no colrorblindness variation, this will have no effect.

My idea is, at the moment all themes do have the colorblind variation, so I did not bother to hide or show the checkbox depending on what classes the actual .css provides, just wait until this is actually required.

Reply:
> "automatic variations"  is slightly better in my opinion

I tried automatic variations to replace red with blue in editor syntax highlighting schemes, and I find the result somewhat unsatisfying. This is why I also kept manually defined red->blue replacements instead of having transformation formulae in the code. I just created a PR #8426 to demonstrate what I mean.

So the issue that bothers me with automatic colorblind transformation, is that the changes nessessarily need to affect a relatively wide range of colors and I have no confidence, that the modification will be for the better in all cases.

Reply:
maybe it's not that bad (automatic colorblind adjustment) , at least the build status colors is modified as expected

before
![image](https://user-images.githubusercontent.com/12046452/91641436-167dea00-ea2d-11ea-8a67-fff0cf8f738d.png)

after
![image](https://user-images.githubusercontent.com/12046452/91641411-dae32000-ea2c-11ea-8f75-c20b12a78680.png)

Reply:
I meant more that if you have a file mytheme.css with a Branch.colorblind and Branch.myvariant, the themes offered are mytheme, mytheme-colorblind, mytheme-myvariant

#8426 is maybe contradicting this view, if there are special handling in other situations too - adding to the changes in the theme itself. This could be handled with a convention (like variant name is colorblind), but a setting is more natural. 
The importance for #8426 (as an extra step) is a separate question. I would expect that the change is welcome. It does not look good to me, but if it makes the UI usable for 1% of users, it could be worth it.